tcms/testruns/templates/testruns/get.html

```diff
@@ -109,13 +109,77 @@ <h2 class="card-pf-title" style="text-align: left">
+                    {% trans 'Test executions' %}
+                </h2>
+                <div class="card-pf-body">
+                    <div id="test-executions-container" class="list-group tree-list-view-pf"></div>
+                </div>
+            </div>
+        </div>
+    </div> <!-- /row -->
+
     <div class="row row-cards-pf">
         <div class="col-xs-12 col-sm-6 col-md-6">
             {% include 'include/attachments.html' %}
         </div>
     </div> <!-- /row -->
 </div>
 
+<template id="test-execution-row">
+    <div class="list-group-item">
+        <div class="list-group-item-header">
+            <div class="list-view-pf-main-info">
+                <div class="list-view-pf-left">
+                    <span class="fa fa-angle-right"></span>
+                </div>
+                <div class="list-view-pf-body">
+                    <div class="list-view-pf-description">
+                        <div class="list-group-item-text">
+                             <span class="test-execution-info"></span>
+                         </div>
+                    </div>
+                    <div class="list-view-pf-additional-info">
+                        <div class="list-view-pf-additional-info-item">
+                            <span class="fa test-execution-automated" data-true="{% trans 'Automated' %}" data-false="{% trans 'Manual' %}"></span>
+                        </div>
+                        <div title="{% trans 'Priority' %}" class="list-view-pf-additional-info-item">
+                            <span class="fa fa-hourglass"></span>
+                            <span class="test-execution-priority"></span>
+                        </div>
+                        <div title="{% trans 'Category' %}" class="list-view-pf-additional-info-item">
+                            <span class="fa fa-tag"></span>
+                            <span class="test-execution-category"></span>
+                        </div>
+                        <div title="{% trans 'Author' %}" class="list-view-pf-additional-info-item">
+                            <span class="fa pficon-user"></span>
+                            <span class="test-execution-asignee"></span>
+                        </div>
+                        <div title="{% trans 'Default tester' %}" class="list-view-pf-additional-info-item">
+                            <span class="fa fa-search"></span>
+                            <span class="test-execution-tester"></span>
+                        </div>
+                        <div title="{% trans 'Bugs' %}" class="list-view-pf-additional-info-item">
+                            <span class="fa fa-bug"></span>
+                            <span class="test-execution-bugs-count"></span>
+                        </div>
+                        <div title="{% trans 'Status' %}" class="list-view-pf-additional-info-item">
+                            <i class="fa fa-2x test-execution-status-icon"></i>
```

Comment:
why fa-2x ? In TestCase page the icon is regular size.

Reply:
actually, this does not matter. `i` has style applied because it is inside `list-view...`. `fa-2x` was just a leftover from something I tried. removed it now.